app/src/sharedTest/java/org/oppia/android/app/onboarding/OnboardingFragmentTest.kt

```diff
@@ -140,6 +140,33 @@ class OnboardingFragmentTest {
     }
   }
 
+  @Test
+  fun testOnboardingFragment_DefaultSlideTitleDescription_hasCorrectContentDescription() {
+    launch(OnboardingActivity::class.java).use {
+      onView(withId(R.id.onboarding_layout)).check(
+        matches(
+          withContentDescription(
+            "Welcome to Oppia! Learn anything you want in an effective and enjoyable way."
```

Comment:
Are we having this string in the strings.xml?
If yes let's use the resources only rather than hardcode string.

Reply:
We are having two strings in strings.xml ```Welcome to Oppia! \u0020``` and ```Learn anything you want in an effective and enjoyable way.``` I tried some approaches of concatenating these two string resources in this Test file but none of them worked ! Should I consider making a new resource ```Welcome to Oppia! Learn anything you want in an effective and enjoyable way.``` or is there any specific approach of concatenating strings in a test file ?

Reply:
Ah, I got it. We are concatenating in the layout file `android:contentDescription="@{viewModel.title} + @{viewModel.description}"`

So, yes, I think it will be better to do the concatenating approach.

What concatenating approaches you had tried and what errors you are getting?

Reply:
Changes done